styleguide/backstop.json

```diff
@@ -232,7 +232,7 @@
       "delay": 5000,
       "hideSelectors": [],
       "removeSelectors": [],
-      "selectors": [],
+      "selectors": [".ama__page-title", ".ama__membership ", ".ama__tools-block", ".ama__promo", ".ama__article-stub", ".ama__lead-news", ".ama__footer"],
```

Comment:
@designerneil when I did this approach during the travis testing the FEDs decided that we should create a scenario for each pattern to test it on it's own instead of just on the page since patterns may be used on multiple pages and while some patterns may be used in Drupal they may not be on the example page. Please separate these out into individual scenarios.